transform.ts

```diff
@@ -260,7 +280,9 @@ function addFunctionTSTypes(
   componentTypes: CollectedTypes
 ) {
   source.forEach((path) => {
-    const typeName = createInterface(path, componentTypes)
+    const typeName = options.preferTypeAliases
+      ? createTypeAlias(path, componentTypes)
```

Comment:
Let's just rename `createInterface` to something more generic and do this as a conditional in that function. Don't want all the duplicate code this currently has.

Reply:
Addressed